src/db/comments.ts

```diff
@@ -0,0 +1,26 @@
+import { relations } from 'drizzle-orm';
+import { pgTable, timestamp, uuid, varchar } from 'drizzle-orm/pg-core';
+import { questionOptions } from './questionOptions';
+import { users } from './users';
+
+export const comments = pgTable('comments', {
+  id: uuid('id').primaryKey().defaultRandom(),
+  userId: uuid('user_id').references(() => users.id),
+  questionOptionId: uuid('question_option_id').references(() => questionOptions.id),
+  comment: varchar('comment').notNull(),
```

Comment:
i would rename this to either `value` or `text` or `comment_value`

Reply:
Good point! Called it `value` as this is consitent with the naming of the calue column in the `registrationData` table.